app/services/search_service.rb

```diff
@@ -182,4 +182,29 @@ def textSearch_questions(srchString)
     end
     sresult
   end
+
+  def search_nearbyLocations(srchString)
```

Comment:
This is looking great! Could we call this `nearbyNodes`?

Reply:
doing changes now